### DISCUSSION TITLE
Very large file size after applying effects

### DISCUSSION THREAD

**Discussion by bnaseath** [audacity/audacity#4660] [General]
I don't see this reported anywhere else. I just installed the new version 3.3.1. I recorded a number of files, the largest is over an hour at 770 MB. I then applied the following effects in order and saved the file as a new file: Filter Curve EQ, Compressor, Loudness Normalization, Limiter, De-Clicker (add on), ACX-Check (add on). Afterward, I saved the file as a new file and waited several minutes and then closed audacity. The new file was 48 GB. I repeated this on 5 files and each one increased proportionately.

**Comment by crsib** [comment]
When you apply effect destructively Audacity has to keep the old data so the action can be undone. Closing Audacity will discard the history and (most likely) will trigger the project compaction.

This was the case before the unified project format was introduced as well, but:
1. It was less noticeable, as the growth happened to the data folder.
2. There was no need for "compaction", the no longer needed files were just removed from the disk.
